tests/test_secrets_awssm.py

```diff
@@ -23,7 +23,11 @@
 import boto3
 import pytest
 
-from moto import mock_secretsmanager
+
+try:
+    from moto import mock_aws
+except ImportError:
+    from moto import mock_secretsmanager as mock_aws
```

Comment:
```suggestion
from moto import mock_aws
```

If this were user code, I would be all for it, but this is just _our_ tests, so there's really no reason to keep compatibility around? Because it means that we'd either have to add another test environment or carry around untested code which the latter seems bad and the former doesn't seem worth it – am I missing something?

Reply:
Nope, I just wasn't sure which way you wanted to go, let me sort this out and I'll push it up presently.